cassandra/util.py

```diff
@@ -555,3 +555,76 @@ def _intersect(self, other):
                     if item in other:
                         isect.add(item)
             return isect
+
+from collections import Mapping
+import six
+from six.moves import cPickle
+
+class OrderedMap(Mapping):
+    '''
+    An ordered map that accepts non-hashable types for keys.
```

Comment:
I would mention that it's not mutable as well.  Also, it would be good to include this class in the API docs.